src/main/java/org/springframework/data/jpa/repository/query/StringQuery.java

```diff
@@ -163,6 +189,13 @@ boolean isDefaultProjection() {
 			PARAMETER_BINDING_PATTERN = Pattern.compile(builder.toString(), CASE_INSENSITIVE);
 		}
 
+		static List<ParameterFinds> parseParameters(String query) {
```

Comment:
Is that intended? Ignoring the parameters looks weird.

Reply:
That was a leftover from a different approach. :face_palm:  Removed it.